src/resources/createResourceRoutes.js

```diff
@@ -101,6 +101,7 @@ const ColumnWrapper = ({ list, details, create, ...props }) => {
 
   const listComponent = React.cloneElement(list, {
     ...elementListProps,
+    rawResourceType: props.resourceType,
```

Comment:
We have a resourceType, the same in `elementListProps`, so we really need the additional property for this?